src/main/java/com/example/Calculator.java

```diff
@@ -1,7 +1,24 @@
 package com.example;
 
+import java.io.FileWriter;   // Import the FileWriter class
+import java.io.IOException;  // Import the IOException class to handle errors
+
 public class Calculator {
     public int add(int a, int b) {
         return a + b;
     }
-}
\ No newline at end of file
+
+    public int subtract(int a, int b) {
+        try {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *myWriter*. Currently, there are execution paths that do not contain closure statements. Either a) close *myWriter* in a try-finally block or b) close the resource by declaring *myWriter* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).